recipes/rgbcx/all/conanfile.py

```diff
@@ -20,13 +19,13 @@ def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = glob.glob('bc7enc-*/')[0]
         os.rename(extracted_dir, self._source_subfolder)
-        self._patch_sources()
 
     def _patch_sources(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "rgbcx.h"),
             "#include <stdlib.h>", "#include <stdlib.h>\n#include <string.h>")
 
     def package(self):
+        self._patch_sources()
```

Comment:
```suggestion
    def build(self):
        tools.replace_in_file(os.path.join(self._source_subfolder, "rgbcx.h"),
                              "#include <stdlib.h>",
                              "#include <stdlib.h>\n#include <string.h>")

    def package(self):
```

Reply:
Yep, I would have to agree... https://github.com/conan-io/conan-center-index/pull/3271#pullrequestreview-512300106